src/Umbraco.Core/Persistence/Migrations/Upgrades/TargetVersionSevenThreeZero/AddRelationTypeForDocumentOnDelete.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Models.Rdbms;
+
+namespace Umbraco.Core.Persistence.Migrations.Upgrades.TargetVersionSevenThreeZero
+{
+    [Migration("7.3.0", 0, GlobalSettings.UmbracoMigrationName)]
+    public class AddRelationTypeForDocumentOnDelete : MigrationBase
+    {
+        public override void Up()
+        {
+            Execute.Code(AddRelationType);
+        }
+
+        public static string AddRelationType(Database database)
+        {
```

Comment:
As this is inserting a new entry I'm pretty sure this way (without setting the id) is perfectly fine. Its how I would do it :wink: